docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,501 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4dd46d67",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "example_issue_idx = issue_idx[10]\n",
+    "\n",
+    "# create a class_labels dictionary to pass in and see labels in the legend\n",
+    "class_names = {\"0\": \"car\", \"1\": \"chair\", \"2\": \"cup\", \"3\":\"person\", \"4\": \"traffic light\"}\n",
+    "\n",
+    "label = labels[example_issue_idx]\n",
+    "prediction = predictions[example_issue_idx]\n",
+    "score = scores[example_issue_idx]\n",
+    "image_path = IMAGE_PATH + label['seg_map']\n",
+    "\n",
+    "print(image_path, '| idx', example_issue_idx , '| label quality score:', score, '| is issue: True')\n",
+    "visualize(image_path, label=label, prediction=prediction, class_names=class_names, overlay=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de0d7205",
+   "metadata": {},
+   "source": [
+    "As you can see the given label is marked in a solid red color on the image to the left. This is the provided annotation in which issues are detected in. The model predictions, or the `predicted label` is in blue and overlayed on the image on the right. Each bounding box contains a class number in the top corner (left for given annotations, right for predicted) specific to which class that bounding box was marked as. With our provided `class_names` dictionary we can easily identify which classes were marked in the legend to the right of both images.\n",
+    "\n",
+    "This image has a **low** label quality score and is marked as an error. On closer inspection we notice the annotator missed a car in the back right corner that the model identified. Additionally, the bounding box for the foreground car the woman is in is cut short a bit on the right. \n",
+    "\n",
+    "Notice examples where the predictions and labels are more similar have higher quality scores than those that are missmatched, and are less likeley to be marked as issues and the number of boxes is agnostic to the score.\n",
+    "\n",
+    "Since our model is not perfect, it incorrectly identified the passenger dog as a person however it was still able to identify issues in the scene. Better trained models will lead to better label error detection but you don't need a near perfect model to identify label issues.\n",
+    "\n",
+    "\n",
+    "### Different kinds of label issues identified by ObjectLab\n",
+    "Now lets view the first 5 images in our vaidation dataset that are clearly marked as issues and see what other inconsistencies between the `given` and `predicted` label we can spot."
```

Comment:
```suggestion
    "The visualization depicts the given label (original image annotation which cleanlab identified as problematic) in red on the left and the model-predicted label in blue on the right. Each bounding box contains a class-index number in the top corner indicating which object class that bounding box was annotated/predicted to contain.\n",
    "\n",
    "This image has a **low** label quality score and is marked as an error. On closer inspection we notice the annotator missed a car in the back right corner that the model identified. Additionally, the bounding box for the foreground car the woman is in is cut short a bit on the right. \n",
    "\n",
    "Notice examples where the predictions and labels are more similar have higher quality scores than those that are missmatched, and are less likeley to be marked as issues and the number of boxes is agnostic to the score.\n",
    "\n",
    "Since our model is not perfect, it incorrectly identified the passenger dog as a person however it was still able to identify issues in the scene. Better trained models will lead to better label error detection but you don't need a near perfect model to identify label issues.\n",
    "\n",
    "\n",
    "### Different kinds of label issues identified by ObjectLab\n",
    "Now lets view the first 5 images in our vaidation dataset that are clearly marked as issues and see what other inconsistencies between the `given` and `predicted` label we can spot."
```